articles/models.py

```diff
@@ -0,0 +1,44 @@
+from django.db import models
+from django.contrib.auth.models import User
+from django.utils import timezone
+import uuid
+
+from .mixin import UUIDMixin, TimeStampedMixin
+
+class Article(UUIDMixin, TimeStampedMixin, models.Model):

```

Comment:
suggestion: models.Model здесь уже не нужно
```suggestion
class Article(UUIDMixin, TimeStampedMixin):

```